### DISCUSSION TITLE
Resubmission for Student Benefits (Access free GitHub Education benefits) does not work.

### DISCUSSION THREAD

**Discussion by alcastronic** [community/community#138364] [GitHub Education] [answered]
### Hi everyone,

My student benefits have expired and I was following the process to renew them. To do so I was guided to open https://education.github.com/discount_requests/application. On this page I should be able to resubmit a request for student benefits.
However this does not seem to work. 

I'm on a Linux host and tried with Chromium and Firefox. Since my study program is remote I'm connected with the universities VPN. 
The initial application process was done with the exact same setup,

- My student e-mail is already validated and connected with my GitHub account.
- My University is already filled out in the application form, it did not change and I'm still enrolled in the same program

When I click the button "Continue" in the below screenshot the browser asks for my location.

![image](https://github.com/user-attachments/assets/4fed6109-4301-406d-9a09-a7af6e244e95)

After I granted the browser, on this page, access to my location the button is grayed out and nothing continues to happen.

- In Chromium dev tools Network tab I do not see any connection
- In Chromium dev tools Console tab I do not see any errors.

**Comment by RaphaelRibes** [comment]
I have the same problem, upping this. My bills are due for the 17 I hope it can be fixed before that 🙏

**Comment by itisrude** [comment, marked as answer]
Hey alcastronic,

It sounds like a browser issue or maybe something with the VPN. Since you mentioned that the button gets grayed out after granting location access, a few things you can try:

- Disable your VPN temporarily and try submitting again. Sometimes VPNs can mess with location-based services.

- Clear your browser cache or try using a private/incognito window.

- Check if any location services are blocked: You can check the browser settings to make sure location permissions are fully enabled.

- Try another browser or update the current one just in case it's a version issue.

If none of these work, contacting GitHub support could help too!

**Comment by RaphaelRibes** [comment]
Using Edge worked for me, thank you so much. I never thought I would have to use Edge for something else than downloading a browser.

**Comment by alcastronic** [comment]
It worked for me using MacBook and the Chrome browser. @itisrude thanks for your inputs!

**Comment by AaronPedersen-QUT** [comment]
I had the same issue when trying on 2 different Microsoft Edge accounts.
But then after seeing this post, I tried on Google Chrome, and the next page actually loaded after I pressed "continue" (it worked).

From reading other replies too, it appears, even if you're using one of their listed "supported browsers", merely switching to a different browser type ( Edge <-> Chrome ) fixes this bug/issue.
